README.md

```diff
@@ -212,6 +212,70 @@ with beeline.tracer("my expensive computation"):
     recursive_fib(100)
 ```
 
+## Using Sampler and Presend Hooks
+
+Hooks give you more power over how events are sampled, and which fields are sent to Honeycomb.
+
+### Using a Sampler Hook
+
+Sampler hooks allow you to define custom sampling behavior. For example, let's say you were instrumenting an HTTP app and wanted to record events data for each error, but heavily sample healthy requests. Also, you don't really care about 302 redirects in your app, and you want to drop those. You could define a sampler function like so:
```

Comment:
I would add a note "Note that using a sampler hook replaces the default deterministic sampling and overrides the default sample rate - this is not in addition to the sample rate set on the beeline config." or something like that.